dashboard/app/models/student_work_evaluation_summary.rb

```diff
@@ -16,9 +16,9 @@
 class StudentWorkEvaluationSummary < ApplicationRecord
   # student_work_evaluation_id is the ID of the StudentWorkEvaluation that was summarized
   # student_work_evaluation_summary_id is the ID of the StudentWorkEvaluation that is the summary
-  # For example, if a UserLevelEvaluation is based on the roll-up of multiple UserSkillEvaluations,
+  # For example, if a UserLevelEvaluation is based on the roll-up of multiple UserLevelSkillEvaluations,
   # then student_work_evaluation_id is the ID of one of the UserLevelSkillEvaluations and
   # student_work_evaluation_summary_id is the ID of the UserLevelEvaluation.
-  belongs_to :student_work_evaluation
-  belongs_to :student_work_evaluation_summary, class_name: 'StudentWorkEvaluation'
+  belongs_to :student_work_evaluation, class_name: 'UserLevelSkillEvaluation'
+  belongs_to :student_work_evaluation_summary, class_name: 'UserLevelEvaluation'
```

Comment:
I'm not sure what the conventions are here in a future world where we might _also_ want something like: 
 ```
belongs_to :student_work_evaluation, class_name: 'UserLevelEvaluation'
belongs_to :student_work_evaluation_summary, class_name: 'SectionLevelEvaluation'
```
but I figure we can refactor when/if the time comes... unless someone has strong opinions otherwise.